src/components/vr_module/include/vr_module/socket_channel.h

```diff
@@ -47,6 +47,7 @@ namespace vr_module {
 class SocketChannel : public Channel {
  public:
   SocketChannel();
+  explicit SocketChannel(net::ConnectedSocket *socket);
```

Comment:
Google style alows both of them, but from my point of view star relates to type not to variable name